songpal/containers.py

```diff
@@ -314,18 +317,59 @@ def __str__(self):
         else:
             return "Power off"
 
+@attr.s
+class Zone:
+    """Zone information.
+
+    This implements __bool__() for easy checking if the device is turned on or not.
+    """
+    make = classmethod(make)
+
+    def _convert_title(x):
+        return x.strip()
+
+    def _convert_is_active(x):
+        return True if x == "active" else False
```

Comment:
There are already two instances of this very same code, could you please move `convert_is_active` to the top-level after `convert_to_bool` and get rid of the copies? Same for the title conversion, too!

Reply:
Done